core/src/main/java/org/pentaho/platform/scheduler2/action/SchedulerOutputPathResolver.java

```diff
@@ -177,13 +189,36 @@ private String runAsUser( Callable<String> callable ) {
     return null;
   }
 
-  protected boolean isValidOutputPath( @NonNull String path ) {
```

Comment:
The original `isValidOutputPath` got renamed to `doesFolderExist`. The new one checks for blank value, and calls both the methods `doesFolderExist` and `isPermitted`, and performs logging.